src/plugin/gradle/src/main/kotlin/ConvertWirespecTask.kt

```diff
@@ -50,6 +50,7 @@ abstract class ConvertWirespecTask : BaseWirespecTask() {
         }
         ConverterArguments(
             format = format.get(),
+            preProcessor = { it },
```

Comment:
I don't think preprocessing is a core feature or needed in the converged arguments